src/test/Auth.t.sol

```diff
@@ -61,46 +60,55 @@ contract AuthTest is DSTestPlus {
         mockAuthChild.setAuthority(new MockAuthority(true));
     }
 
-    function testFailSetOwnerAsNonOwner() public {
+    function testSetOwnerAsNonOwner() public {
         mockAuthChild.setOwner(address(0));
+        vm.expectRevert("UNAUTHORIZED");
         mockAuthChild.setOwner(address(0xBEEF));
     }
 
-    function testFailSetAuthorityAsNonOwner() public {
+    function testSetAuthorityAsNonOwner() public {
         mockAuthChild.setOwner(address(0));
+        // Fails since MockAuthChild uses Authority(address(0)) which can't call `canCall` function
+        vm.expectRevert();
```

Comment:
oh huh this made me realize we need a revert string here

https://github.com/Rari-Capital/solmate/blob/9f16db2144cc9a7e2ffc5588d4bf0b66784283bd/src/auth/Auth.sol#L41

Reply:
Added `"UNAUTHORIZED"` (?)